### PR TITLE
Raise an exception when CSV reading fails

### PR DESCRIPTION
**Pull request by waxlamp**
Addresses https://github.com/multinet-app/multinet-client/issues/20

### CODE REVIEW

File: multinet/uploaders/csv.py [hunk 1]
```diff
@@ -91,7 +99,10 @@ def upload(workspace: str, table: str) -> Any:
     # Read the request body into CSV format
     body = decode_data(request.data)
 
-    rows = list(csv.DictReader(StringIO(body)))
+    try:
+        rows = list(csv.DictReader(StringIO(body)))
+    except csv.Error:
+        raise CSVReadError()
```
Comment (AlmightyYakob) [inline multinet/uploaders/csv.py]: Do you have a file what would produce this error? Every time I try it raises other errors instead.

**Comment by AlmightyYakob** [comment]
It seems like this one overlaps with #354? All of the changes there are included in this PR as well.

File: multinet/uploaders/csv.py (hunk 1 above)
Comment (waxlamp) [reply multinet/uploaders/csv.py]: I used a gpg-encrypted file, but I think you can use any binary file (like some random PDF file you have lying around) and it's likely to trigger this.

**Comment by waxlamp** [comment]
> It seems like this one overlaps with #354? All of the changes there are included in this PR as well.

Yeah it looks like this PR and that one are orthogonal. I thought this one depended on that one for some reason, but it's moot now since I've merged #354.

File: multinet/uploaders/csv.py (hunk 1 above)
Comment (AlmightyYakob) [reply multinet/uploaders/csv.py]: Yep that did it, thanks :+1:

**Pull request merged**